tests/sanity/ignore-2.10.txt

```diff
@@ -1 +1,3 @@
 plugins/action/vyos.py action-plugin-docs # base class for deprecated network platform modules using `connection: local`
+plugins/module_utils/network/vyos/config/ospf_interfaces/ospf_interfaces.py compile-2.6!skip
```

Comment:
What's the issue in this file? Are there other dict comprehensions that didn't get fixed?